src/store/cache/analytics/hooks.ts

```diff
@@ -289,132 +255,86 @@ export function fetchTotalStaked(
   }
 }
 
-const getTrailingAPI = (endpoint: string) => async () => {
-  const url = `${endpoint}/v1/metrics/routes/trailing/month`
-  const json = await fetchWithTimeout(url)
-  return {
-    count: json?.data?.count ?? 0,
-    unique_count: json?.data?.unique_count ?? 0
-  } as CountRecord
-}
-
-const getTrailingAPILegacy = (
-  endpoint: string,
-  startTime: number
-) => async () => {
-  const url = `${endpoint}/v1/metrics/routes?bucket_size=century&start_time=${startTime}`
-  const json = await fetchWithTimeout(url)
+const getTrailingAPI = async (nodes: DiscoveryProvider[]) => {
+  const json = await Promise.any(
+    nodes.map(node => {
+      const url = `${node.endpoint}/v1/metrics/aggregates/routes/trailing/month`
```

Comment:
can we just use the selected discprov instead of racing all of these? it's kind of unnecessary load. you can refer to how we do the genre chart for this